datasources/04_ACT_Government.json

```diff
@@ -3,7 +3,220 @@
     {
       "type": "group",
       "name": "ACT Government",
-      "items": [],
+      "items": [
+        {
+            "type": "esri-mapServer-group",
+            "url": "http://data.actmapi.act.gov.au/arcgis/rest/services/actmapi",
+            "name": "All ACT Mapi layers"
```

Comment:
Maybe consider setting `"preserveOrder": true` on the ACT Government group so that this "All" layer stays on the top?